playwright-e2e/tests/dsm/tissueRequestFlow/cmi-tissue-request-flow.spec.ts

```diff
@@ -0,0 +1,122 @@
+import {StudyEnum} from 'dsm/component/navigation/enums/selectStudyNav-enum';
+import {test} from 'fixtures/dsm-fixture';
+import ParticipantListPage from 'dsm/pages/participant-list-page';
+import {AdditionalFilter} from 'dsm/component/filters/sections/search/search-enums';
+import ParticipantPage from 'dsm/pages/participant-page/participant-page';
+import {TabEnum} from 'dsm/component/tabs/enums/tab-enum';
+import OncHistoryTab from 'dsm/component/tabs/onc-history-tab';
+import {
+  OncHistoryInputColumnsEnum,
+  OncHistorySelectRequestEnum
+} from 'dsm/component/tabs/enums/onc-history-input-columns-enum';
+import {expect} from '@playwright/test';
+import {getDate} from 'utils/date-utils';
+
+test.describe('Tissue Request Flow', () => {
+  const studies = [StudyEnum.PANCAN];
+
+  for (const study of studies) {
+    test(`Tissue Request Flow for ${study} study @dsm @feature`, async ({page, request}) => {
+      const participantListPage = await ParticipantListPage.goto(page, study, request);
+      const customizeViewPanel = participantListPage.filters.customizeViewPanel;
+      const searchPanel = participantListPage.filters.searchPanel;
+
+      await test.step('Search for the right participant', async () => {
+        await customizeViewPanel.open();
+        await customizeViewPanel.selectColumns('Medical Record Columns', ['MR Problem']);
+        await customizeViewPanel.selectColumns('Participant - DSM Columns', ['Onc History Created']);
+        await customizeViewPanel.selectColumns('Research Consent Form Columns', ['Your Mailing Address *']);
+
+        await searchPanel.open();
+        await searchPanel.checkboxes('Status', {checkboxValues: ['Enrolled']});
+        await searchPanel.checkboxes('MR Problem', {checkboxValues: ['No']});
+        await searchPanel.dates('Onc History Created', {additionalFilters: [AdditionalFilter.EMPTY]});
+        await searchPanel.text('Your Mailing Address *', {additionalFilters: [AdditionalFilter.NOT_EMPTY]});
+
+        await searchPanel.search();
+      })
+
+      const participantListTable = participantListPage.participantListTable;
+      const participantPage: ParticipantPage = await participantListTable.openParticipantPageAt(0);
+      const oncHistoryTab = await participantPage.clickTab<OncHistoryTab>(TabEnum.ONC_HISTORY);
+      const oncHistoryTable = oncHistoryTab.table;
+
+      await test.step('Update Onc History data - Facility', async () => {
```

Comment:
Format indentation is wrong in this file.

Reply:
What do you mean?

Reply:
Tab size/whitespaces on new line. File looks okay now but I saw 4 whitespaces before.